docs/snippets/metrics/basicUsage.ts

```diff
@@ -3,5 +3,5 @@ import { Metrics } from '@aws-lambda-powertools/metrics';
 const metrics = new Metrics({ namespace: 'serverlessAirline', serviceName: 'orders' });
 
 export const handler = async (_event, _context): Promise<void> => {
-    // ...
+  // ...
```

Comment:
Let's remove the empty comment (`// ...`) and add a command, for instance adding a metric or something like that (you can copy one from the other examples that add a `successful booking` metric)